tests/providers/openlineage/plugins/test_openlineage_adapter.py

```diff
@@ -240,6 +389,63 @@ def test_emit_failed_event(mock_stats_incr, mock_stats_timer):
     mock_stats_timer.assert_called_with("ol.emit.attempts")
 
 
+@mock.patch("airflow.providers.openlineage.plugins.adapter.Stats.timer")
+@mock.patch("airflow.providers.openlineage.plugins.adapter.Stats.incr")
+def test_emit_failed_event_with_additional_information(mock_stats_incr, mock_stats_timer):
+    client = MagicMock()
+    adapter = OpenLineageAdapter(client)
+
+    run_id = str(uuid.uuid4())
+    event_time = datetime.datetime.now().isoformat()
+    adapter.fail_task(
+        run_id=run_id,
+        end_time=event_time,
+        parent_job_name="parent_job_name",
+        parent_run_id="parent_run_id",
+        job_name="job",
+        task=OperatorLineage(
+            inputs=[Dataset(namespace="bigquery", name="a.b.c"), Dataset(namespace="bigquery", name="x.y.z")],
+            outputs=[Dataset(namespace="gs://bucket", name="exported_folder")],
+            run_facets={"sql1": SqlJobFacet(query="SELECT 1;")},
```

Comment:
Why do you attach `JobFacet` to run facets?

Reply:
I wanted to attach any sample Facet that will not interfere with automatically added facets. I will change that to use Run typ facet.